src/pages/colorstyle.tsx

```diff
@@ -0,0 +1,76 @@
+export const colorstyles = {
```

Comment:
Nice abstraction 👍  Good choice to pull this out into its own file because otherwise, it would clutter the component

Reply:
Thank you!